projects/1.18/assets/additional-lanterns/additionallanterns/lang/zh_cn.json

```
@@ -0,0 +1,363 @@
+{
+  "block.additionallanterns.green_blackstone_lantern": "绿色黑石灯笼",
+  "block.additionallanterns.cyan_smooth_stone_lantern": "青色平滑石灯笼",
+  "block.additionallanterns.yellow_warped_lantern": "黄色诡异木灯笼",
+  "block.additionallanterns.white_purpur_lantern": "白色紫珀灯笼",
+  "block.additionallanterns.gray_crimson_lantern": "灰色绯红木灯笼",
+  "block.additionallanterns.end_stone_chain": "末地石锁链",
+  "block.additionallanterns.magenta_obsidian_lantern": "品红色黑曜石灯笼",
+  "block.additionallanterns.blue_obsidian_lantern": "蓝色黑曜石灯笼",
+  "block.additionallanterns.orange_normal_sandstone_lantern": "橙色砂岩灯笼",
+  "block.additionallanterns.light_gray_red_sandstone_lantern": "淡灰色灯笼",
+  "block.additionallanterns.red_smooth_stone_lantern": "红色平滑石灯笼",
+  "block.additionallanterns.red_purpur_lantern": "红色紫珀灯笼",
+  "block.additionallanterns.lime_bricks_lantern": "黄绿色砖块灯笼",
+  "block.additionallanterns.cyan_normal_lantern": "青色灯笼",
+  "block.additionallanterns.purple_normal_lantern": "紫色灯笼",
+  "block.additionallanterns.green_bricks_lantern": "绿色砖块灯笼",
+  "block.additionallanterns.blue_purpur_lantern": "蓝色紫珀灯笼",
+  "block.additionallanterns.magenta_red_sandstone_lantern": "品红色红砂岩灯笼",
+  "block.additionallanterns.blue_prismarine_lantern": "蓝色海晶石灯笼",
+  "block.additionallanterns.white_end_stone_lantern": "白色末地石灯笼",
+  "block.additionallanterns.white_quartz_lantern": "白色石英灯笼",
+  "block.additionallanterns.blue_quartz_lantern": "蓝色石英灯笼",
+  "block.additionallanterns.andesite_chain": "安山岩锁链",
+  "block.additionallanterns.green_red_nether_bricks_lantern": "绿色红下界砖灯笼",
```

原版译名为`红色下界砖块`
虽然`绿色红色下界砖块灯笼`和`红色红色下界砖块灯笼`很怪，但只能这么译了
```suggestion
  "block.additionallanterns.green_red_nether_bricks_lantern": "绿色红色下界砖块灯笼",
```